Dockerfile.test-infra

```diff
@@ -2,6 +2,9 @@ FROM quay.io/ocpmetal/assisted-service:latest AS service
 
 FROM docker.io/centos:8
 
+ARG client_download_repo=https://openshift-release-artifacts.svc.ci.openshift.org/4.6.0-fc.9
```

Comment:
Maybe just the version? With a default value

Reply:
we don't know when we will need to update oc/kubectl, it might be that we will need to pull them from some nightly, so i prefer that user(us) specify the explicit repo and file